crates/bevy_ui/src/ui_node.rs

```diff
@@ -398,6 +430,40 @@ impl Default for AlignItems {
     }
 }
 
+/// How items are aligned according to the cross axis
```

Comment:
Both this doc-comment as well as the doc-comment on `AlignItems` state 

> How items are aligned according to the cross axis

We should probably have some comment on both explaining the difference between them and / or when to use them?